### PR TITLE
Transcripts/decode html entities

### PR DESCRIPTION
**Pull request by SergioEstevao**
| 📘 Part of: #1848  |  
|:---:|

Fixes # 

Update Transcripts for the html format to decode HTML entities

## To test

1. Start the app
2. Open a podcast with Transcripts in the HTML format using html entities. Ex: https://pca.st/episode/1cb156bc-c05c-4e20-ada7-c3fb13c4d380
3. Play an episode of the podcast
4. Open the full screen player
5. Tap on Transcripts
6. Check if the transcript does not show any html entitities.

## Checklist

- [x] I have considered if this change warrants user-facing release notes and have added them to `CHANGELOG.md` if necessary.
- [x] I have considered adding unit tests for my changes.
- [x] I have updated (or requested that someone edit) [the spreadsheet](https://docs.google.com/spreadsheets/d/107jqrutZhU0fVZJ19SBqxxVKbV2NWSdQC9MFYdLiAxc/edit?usp=sharing) to reflect any new or changed analytics.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/Automattic/pocket-casts-ios/milestone/59">7.73 ❄️</a>. The due date for this milestone has already passed.<br />
Please assign it to a milestone with a later deadline or check whether the release for this milestone has already been finished.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@
 - Fix Sharing with AirDrop fails [#2165](https://github.com/Automattic/pocket-casts-ios/issues/2165)
 - Transcripts: Refactor scroll to range textKit code [#2155](https://github.com/Automattic/pocket-casts-ios/pull/2155)
 - Fix sorting of Episode list on WatchOS [#2172](https://github.com/Automattic/pocket-casts-ios/pull/2172)
+- Fix crash on shelf actions [#2174](https://github.com/Automattic/pocket-casts-ios/pull/2174)
```
Comment (SergioEstevao) [inline CHANGELOG.md:10]: Using this PR to add missing changelog entry

Comment (danielebogo) [reply CHANGELOG.md:10]: Thanks!

**Review by danielebogo — APPROVED** [review]
Tested and worked on that episode

Simulator Screenshot - iPhone 16 - 2024-09-23 at 14 31 01

**Pull request merged**